tests/ext/django/app/settings.py

```diff
@@ -44,7 +44,6 @@
     'django.middleware.common.CommonMiddleware',
     'django.middleware.csrf.CsrfViewMiddleware',
     'django.contrib.auth.middleware.AuthenticationMiddleware',
```

Comment:
Why did we remove this 'django.contrib.auth.middleware.SessionAuthenticationMiddleware' ?

Reply:
> The SessionAuthenticationMiddleware class is removed. It provided no functionality since session authentication is unconditionally enabled in Django 1.10.

https://docs.djangoproject.com/en/2.0/releases/2.0/#miscellaneous